uwblueprint/focus-on-nature
# Feat/add personal info questions step1

## Notion ticket link

[Registrant Experience: Fetching questions for personal info step](https://www.notion.so/uwblueprintexecs/Task-Board-F22-9e03bf9f7c4343f09a8a5c16dc48a12a?p=52b4d2b126224c5dbe90de7a428cc6a9&pm=c)



## Implementation description
* This PR assumes only the primary contact has emergencyContact questions 



## Steps to test
1. Go to http://localhost:3000/register/camp/63b01ddec71ee220ef84daeb and register a few campers -> you should see emergency contact info questions + personalInfo questions



## What should reviewers focus on?
* 


## Checklist
